index.html

```diff
@@ -42,6 +42,182 @@ <h2>I'm a Software Developer</h2>
                 <img src="assets/Group.png" alt="bg" class="bg_img">
             </div>
         </section>
+
+        <section id="works" class="work_section">
+            <h2 class="title">My Recent Works</h2>
+            <img src="assets/Vector.png" alt="vector">
+
+            <div class="work1 flex_column">
+                <img src="assets/Img Placeholder.png" alt="placeholder">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately 
+                    personalized reads; no accounts 
+                    or sign-ups required. has been 
+                    the industry's standard dummy 
+                    text ever since the 1500s, when 
+                    an unknown printer took a 
+                    standard dummy text.</p>
+                <ul class="work1_list flex">
+                    <li><button class="work_button">Css</button></li>
+                    <li><button class="work_button">HTML</button></li>
+                    <li><button class="work_button">Bootstrap</button></li>
+                    <li><button class="work_button">Ruby</button></li>
+                </ul>
+                <button class="button_enabled">See project</button>
+            </div>    
+            
+            <div class="works">
+                <div class="work">
+                    <h2>Profesional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately 
+                        personalized reads; no accounts 
+                        or sign-ups required. Has been 
+                        the industry's standard.</p>
+                    <div class="">
+                        <ul class="work_list flex">
+                            <li><button class="works_button">HTML</button></li>
+                            <li><button class="works_button">Bootstrap</button></li>
+                            <li><button class="works_button">Ruby</button></li>
+                        </ul>
+                    </div>
+                        <button class="see_project button_enabled">See project</button>
+                </div>
+
+                <div class="work">
+                    <h2>Profesional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately 
+                        personalized reads; no accounts 
+                        or sign-ups required. Has been 
+                        the industry's standard.</p>
+                    <div class="">
+                        <ul class="work_list flex">
+                            <li><button class="works_button">HTML</button></li>
+                            <li><button class="works_button">Bootstrap</button></li>
+                            <li><button class="works_button">Ruby</button></li>
+                        </ul>
+                    </div>
+                        <button class="see_project button_enabled">See project</button>
```

Comment:
- [ ] kindly consider putting into consideration that the all the buttons follow the button styling: Active, Hover and Pressed this will give a best user experience

Reply:
In the Figma design of this template they are only using the Active button, that's why I created mines